src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -63,6 +63,7 @@ internal partial class CurlHandler : HttpMessageHandler
         private bool _useCookie = HttpHandlerDefaults.DefaultUseCookies;
         private bool _automaticRedirection = HttpHandlerDefaults.DefaultAutomaticRedirection;
         private int _maxAutomaticRedirections = HttpHandlerDefaults.DefaultMaxAutomaticRedirections;
+        private bool _useDefaultCredentials = false;
```

Comment:
WinHttpHandler does not have the corresponding property - On windows, ```httpClientHandler.UseDefaultCredentials``` is computed based on whether serverCredentials is the same instance as CredentialCache.DefaultCredentials or not. 
The reason for introducing the field gives us a little more freedom on how we achieve the functionality of getting the current logged in user name and password in the managed layer.